editor/code_editor.cpp

```diff
@@ -332,6 +332,24 @@ void FindReplaceBar::_update_results_count() {
 	}
 }
 
+void FindReplaceBar::_update_matches_label() {
+
+	if (results_count > 0) {
+		matches_label->show();
+
+		matches_label->add_color_override("font_color", Color(1, 1, 1));
+		matches_label->set_text(vformat(TTR("%d match(es)."), results_count));
+	} else if (search_text->get_text().empty()) {
+
+		matches_label->hide();
+	} else {
+		matches_label->show();
+
+		matches_label->add_color_override("font_color", EditorNode::get_singleton()->get_gui_base()->get_color("error_color", "Editor"));
+		matches_label->set_text(TTR("No Matches"));
```

Comment:
I think the M should likely be lowercase here, it's not a button/title label but a "normal" sentence.